packages/domain-picker/src/domain-picker/suggestion-item.tsx

```diff
@@ -92,7 +92,7 @@ const DomainPickerSuggestionItem: React.FC< Props > = ( {
 	const { __ } = useI18n();
 	const localizeUrl = useLocalizeUrl();
 
-	const isMobile = useViewportMatch( 'small', '<' );
+	const isMobile = ! useViewportMatch( 'small', '>=' );
```

Comment:
This prevents a small bug where, when the viewport is 600px wide, this check would previously return `true`, while the CSS for mobile styles would only be applied below 599px